integrate/simpsons_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright ©2019 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package integrate
+
+import (
+	"math"
+	"testing"
+
+	"gonum.org/v1/gonum/floats"
+)
+
+func TestSimpsons(t *testing.T) {
+	const N = 1e6
```

Comment:
This is unnecessarily large.

Reply:
I don't think so. I think that's a suitable range.

Reply:
Why? Using less points makes it harder to pass the test and so makes it more reliable. Also with less points the test finishes faster.